adafruit_as7341.py

```diff
@@ -329,82 +354,82 @@ def all_channels(self):
         return reads
 
     @property
-    def channel_415nm(self):
+    def channel_415nm(self) -> int:
         """The current reading for the 415nm band"""
         self._configure_f1_f4()
         return self._channel_0_data
 
     @property
-    def channel_445nm(self):
+    def channel_445nm(self) -> int:
         """The current reading for the 445nm band"""
         self._configure_f1_f4()
         return self._channel_1_data
 
     @property
-    def channel_480nm(self):
+    def channel_480nm(self) -> int:
         """The current reading for the 480nm band"""
         self._configure_f1_f4()
         return self._channel_2_data
 
     @property
-    def channel_515nm(self):
+    def channel_515nm(self) -> int:
         """The current reading for the 515nm band"""
         self._configure_f1_f4()
         return self._channel_3_data
 
     @property
-    def channel_555nm(self):
+    def channel_555nm(self) -> int:
         """The current reading for the 555nm band"""
         self._configure_f5_f8()
         return self._channel_0_data
 
     @property
-    def channel_590nm(self):
+    def channel_590nm(self) -> int:
         """The current reading for the 590nm band"""
         self._configure_f5_f8()
         return self._channel_1_data
 
     @property
-    def channel_630nm(self):
+    def channel_630nm(self) -> int:
         """The current reading for the 630nm band"""
         self._configure_f5_f8()
         return self._channel_2_data
 
     @property
-    def channel_680nm(self):
+    def channel_680nm(self) -> int:
         """The current reading for the 680nm band"""
         self._configure_f5_f8()
         return self._channel_3_data
 
     @property
-    def channel_clear(self):
+    def channel_clear(self) -> int:
         """The current reading for the clear sensor"""
         self._configure_f5_f8()
         return self._channel_4_data
 
     @property
-    def channel_nir(self):
+    def channel_nir(self) -> int:
         """The current reading for the NIR (near-IR) sensor"""
         self._configure_f5_f8()
         return self._channel_5_data
 
-    def _wait_for_data(self, timeout=1.0):
+    def _wait_for_data(self, timeout: Optional[float] = 1.0) -> None:
```

Comment:
Looks like it can't take `None` so `timeout` argument should be typed to `float`!

Reply:
If I understand correctly this one should be:
` timeout: float = 1.0`

Reply:
Adjusted - thanks!